frontend/src/entry/model/mirroredModels/ReleasesSelector.tsx

```diff
@@ -15,49 +15,48 @@ type ReleasesSelectorProps = {
 
 export default function ReleaseSelector({ model, selectedReleases, setSelectedReleases }: ReleasesSelectorProps) {
   const [checked, setChecked] = useState(false)
-  const [errorMessage, setErrorMessage] = useState('')
   const { releases, isReleasesLoading, isReleasesError } = useGetReleasesForModelId(model.id)
-  if (isReleasesLoading) {
-    return <Loading />
-  }
-  if (isReleasesError) {
-    setErrorMessage(isReleasesError.info.message)
-  }
 
-  const handleRemoveRelease = async (removedRelease: ReleaseInterface) => {
-    setChecked(false)
-    setSelectedReleases(selectedReleases.filter((release) => release.semver !== removedRelease.semver))
-  }
+  const selectedReleasesDisplay = useMemo(() => {
+    const handleRemoveRelease = async (removedRelease: ReleaseInterface) => {
```

Comment:
Again sorry for giving you a bad snippet for this one. This function should be outside of the `useMemo` and will need to be wrapped with `useCallback` to make it a stable dependency for the `useMemo`.